x-pack/plugins/elastic_assistant/server/lib/langchain/execute_custom_llm_chain/index.ts

```diff
@@ -88,21 +107,95 @@ export const callAgentExecutor = async ({
     request,
     size,
   };
-  const tools: Tool[] = assistantTools.flatMap((tool) => tool.getTool(assistantToolParams) ?? []);
+  const tools: ToolInterface[] = assistantTools.flatMap(
+    (tool) => tool.getTool(assistantToolParams) ?? []
+  );
 
   logger.debug(`applicable tools: ${JSON.stringify(tools.map((t) => t.name).join(', '), null, 2)}`);
 
-  const executor = await initializeAgentExecutorWithOptions(tools, llm, {
-    agentType: 'chat-conversational-react-description',
-    memory,
-    verbose: false,
-  });
+  // isStream check is not on agentType alone because typescript doesn't like
+  const executor = isStream
+    ? await initializeAgentExecutorWithOptions(tools, llm, {
+        agentType: 'openai-functions',
+        memory,
+        verbose: true,
```

Comment:
```suggestion
        verbose: false,
```

Been testing a bit to see what the different log levels look like. I think we'll want this off by default as it clutters the logs and doesn't include the time, application prefix, etc:

<img width="1092" alt="image" src="https://github.com/elastic/kibana/assets/2946766/d739caa6-c052-41aa-a377-051295435a98">

Reply:
yes meant to do that before requesting review, good catch